server/tinylicious/src/routes/storage/git/blobs.ts

```diff
@@ -15,7 +15,7 @@ export async function createBlob(
     authorization: string,
     body: ICreateBlobParams,
 ): Promise<ICreateBlobResponse> {
-    const buffer = Buffer.from(body.content, body.encoding);
+    const buffer = Buffer.from(body.content, body.encoding as BufferEncoding);
```

Comment:
Cast required w/new node .d.ts -- ICreateBlobParams.encoding should probably be retyped as BufferEncoding.  Will follow up.